guides/creating-new-components.md

```diff
@@ -0,0 +1,76 @@
+# Creating a New Component
+
+###### Last updated April 12, 2021
+
+:::
+
+##### Overview
+
+The Cashmere library is heavily built on it's component-based architecture. By adding new and exciting components to this library you are only enriching what it has to offer to it's developers. Here are some things to remember when creating new components.
+
+:::
+
+:::
+
+##### Creating the Component Manually
+
+<!-- We will be using our banner component as an example of how to create a new component. -->
+
+1. Start by creating a new folder for your component in ```projects/cashmere/src/lib```
+
+This folder should contain the following files:
+
+* "component-name".directive.ts
+
+- "component-name".component.spec.ts
+
+* "component-name".component.ts
+
+- "component-name".html
+
+* "component-name".module.ts - this file exports all the public properties of your component.
+
+- "component-name".scss - this will be used to style your component. For more on how to follow our styling conventions, [click here](http://localhost:4200/foundations/color)
```

Comment:
Could we link here instead? https://cashmere.healthcatalyst.net/web/guides/using-customizing-components?section=style-guidelines-for-new-components&selected=using-customizing-components

This explains the pattern we recommend using sass mixins instead of css rules directly.